src/evo/assetlocktx.cpp

```diff
@@ -140,7 +140,8 @@ bool CAssetUnlockPayload::VerifySig(const llmq::CQuorumManager& qman, const uint
     const auto quorum = qman.GetQuorum(llmqType, quorumHash);
     // quorum must be valid at this point. Let's check and throw error just in case
     if (!quorum) {
-        return state.Invalid(TxValidationResult::TX_CONSENSUS, "internal-error");
+        LogPrint(BCLog::CREDITPOOL, "No quorum for found for hash=%s\n", quorumHash.ToString());
```

Comment:
Log status here should be not CREDITPOOL, but ERROR, because it's not expected failure
also typo "for found for"

```diff
-LogPrint(BCLog::CREDITPOOL, "No quorum for found for hash=%s\n", quorumHash.ToString());
+LogPrintf("ERROR! No quorum for CreditPool found for hash=%s\n", quorumHash.ToString());

Reply:
Resolved in latest push!